packages/react-components/react-badge/src/components/CounterBadge/CounterBadge.tsx

```diff
@@ -4,14 +4,19 @@ import { useCounterBadgeStyles_unstable } from './useCounterBadgeStyles';
 import { renderBadge_unstable } from '../Badge/index';
 import type { CounterBadgeProps } from './CounterBadge.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
```

Comment:
_nit:_ Keep `import type` statements at the bottom

```suggestion
import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
import { renderBadge_unstable } from '../Badge/index';
import type { CounterBadgeProps } from './CounterBadge.types';
import type { ForwardRefComponent } from '@fluentui/react-utilities';
```